weblab/repocache/migrations/0016_auto_20200107_1710.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.23 on 2020-01-07 17:10
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('repocache', '0015_auto_20191128_1331'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='cachedmodelversion',
+            name='master_filename',
+            field=models.TextField(default=' ', help_text='Master filename'),
```

Comment:
You might need to run `makemigrations` again - the default in `models.py` has changed?

Reply:
yes running it again produced a second file, thanks. I wonder why its not possible to add a field and set defaults in a single migration?

Reply:
It can do both at once. I think here we changed the definition of the field after the initial migration was created.